resources/public/api/conf/1.0/application.raml

```diff
@@ -37,4 +37,6 @@ uses:
         type: string
         example: TC663795B
     /list:
-      get: !include list.raml
\ No newline at end of file
+      get: !include list.raml
+    /{businessId}:
+      get: !include retrieve.raml
```

Comment:
On The look of the spec it looks like it should be business-details/nino etc. instead of business/details/nino etc.

Reply:
The BusinessId Path parameter is missing